### ISSUE TITLE
Use of 'hiera.yaml' version 3 is deprecated

### ISSUE THREAD

**Issue by ponsfrilus** [epfl-sti/puppet.epfl_sso#13]
Running the code
```puppet
puppet apply -e "class { 'epfl_sso':
     allowed_users_and_groups => 'username (groupname)',
     auth_source => 'scoldap',
     directory_source => 'scoldap'
}"
```
raises the following error:
```
Warning: /etc/puppetlabs/puppet/hiera.yaml: Use of 'hiera.yaml' version 3 is deprecated. It should be converted to version 5
   (in /etc/puppetlabs/puppet/hiera.yaml)
Error: Evaluation Error: Error while evaluating a Function Call, Need version 3.x of Puppet. at /etc/puppetlabs/code/environments/production/modules/epfl_sso/manifests/init.pp:61:5 on node xxx.yyy.zz
```
On `puppet --version`: `4.9.4`. See the following line of [init.pp](https://github.com/epfl-sti/puppet.epfl_sso/blob/master/manifests/init.pp#L61).

**Comment by domq** [comment]
Weird — There is no hiera business in epfl_sso as far as I can tell.
```
ag -r hiera /etc/puppet/code/modules/epfl_sso  # Turns up nothing
```
Can you please move your `/etc/puppetlabs/puppet/hiera.yaml` out of the way, and try again?

**Comment by domq** [comment]
Corrected in [2f6c5ce9b7b of epfl-sti/quirks](https://github.com/epfl-sti/puppet.quirks/commit/2f6c5ce9b7b5b51344b360396bd8684f713a65ac)
